Src/BootCamp.Chapter/UserInteraction.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    public static class UserInteraction
+    {
+        private const string characterMask = "\u263A";
+        private const string backspace = "\b \b";
+        private static readonly Credentials newCredentials = new Credentials();
+
+        public static void DisplayMainMenu()
+        {
+            ConsoleInit();
+            DisplayHeader("Main menu");
+
+            Console.WriteLine("(1) Login");
+            Console.WriteLine("(2) Register");
+            Console.WriteLine();
+            Console.WriteLine("(0) Exit!");
+
+            try
+            {
+                ChooseMenu();
+            }
+            catch (InvalidCredentialsDbFile ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            catch (InvalidNameException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            catch (InvalidPasswordException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            catch (UserAllreadyExistsException ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+        }
+
+        private static void ChooseMenu()
+        {
+            ConsoleKey pressedKey = Console.ReadKey(true).Key;
```

Comment:
if you're going to read the specific key, probably should support numpad numbers also :)

Reply:
Last time I implemented a console menu and used both type of keys Codacy complained about Cyclomatic Complexity. I think it should not be a problem now as there are less options to choose from.